### PR TITLE
feat: detach address from building unit

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/BuildingRegistry/Building/Building_BuildingUnit.cs [hunk 1]
```diff
@@ -179,6 +179,15 @@ public void DeregulateBuildingUnit(BuildingUnitPersistentLocalId buildingUnitPer
                 .AttachAddress(addressPersistentLocalId, _addresses);
         }
 
+        public void DetachAddressToBuildingUnit(
```
Comment (jvandaal) [inline src/BuildingRegistry/Building/Building_BuildingUnit.cs:182]: DetachAddressToBuildingUnit should be DetachAddress**From**BuildingUnit

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/DetachAddressFromBuildingUnitLambdaHandler.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    using Abstractions.Validation;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Exceptions;
+    using Microsoft.Extensions.Configuration;
+    using Requests.BuildingUnit;
+    using TicketingService.Abstractions;
+
+    public sealed class
+        DetachAddressFromBuildingUnitLambdaHandler : BuildingUnitLambdaHandler<
+            DetachAddressFromBuildingUnitLambdaRequest>
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/DetachAddressFromBuildingUnitLambdaHandler.cs:20]: one or two lines please

**Comment by emalfroy** [comment]
Todo: Controllers aanmaken

**Review by ArneD — CHANGES_REQUESTED** [review]

**Pull request merged**